clang/include/clang/Basic/Builtins.h

```diff
@@ -408,7 +408,9 @@ class Context {
 
   unsigned getRequiredVectorWidth(unsigned ID) const;
 
-  /// Return true if builtin ID belongs to AuxTarget.
+  /// Return true if builtin ID belongs to only the AuxTarget.
```

Comment:
Perhaps, "Return true if the builtin ID belongs exclusively to the AuxTarget, and false if it belongs to both primary and aux target, or neither."

WDYT?

Reply:
sure, applied this in latest commit.